recipe/build.sh

```diff
@@ -1,26 +1,23 @@
 #!/bin/bash
 
-EXTRA_ARGS="CUDARTLIB=\"cudart_static\""
```

Comment:
I notice this is gone, any reason to change it? Note about the long discussion before we switched to use static cudart (#78, #80, and issues/PRs linked therein).

Reply:
It's static by default: https://github.com/NVIDIA/nccl/blob/2ea4ee94bfb04c886c79ccae60ac9961000fdee2/src/Makefile#L30

Reply:
keeping it in is of course fine too in case that changes.

Reply:
Enabling conda-build's overlinking checks in PR: https://github.com/conda-forge/nccl-feedstock/pull/118

This should catch dynamic linking should it happen accidentally